src/shape/vertex.js

```diff
@@ -803,33 +802,21 @@ function vertex(p5, fn){
    * @param  {Number} x4
    * @param  {Number} y4
    * @param  {Number} z4 z-coordinate of the anchor point.
-   * @chainable
    */
   fn.bezierVertex = function(...args) {
-    p5._validateParameters('bezierVertex', args);
-    if (this._renderer.isP3D) {
-      this._renderer.bezierVertex(...args);
-    } else {
-      if (vertices.length === 0) {
-        p5._friendlyError(
-          'vertex() must be used once before calling bezierVertex()',
-          'bezierVertex'
-        );
-      } else {
-        isBezier = true;
-        const vert = [];
-        for (let i = 0; i < args.length; i++) {
-          vert[i] = args[i];
-        }
-        vert.isVert = false;
-        if (isContour) {
-          contourVertices.push(vert);
-        } else {
-          vertices.push(vert);
-        }
+    if (args.length === 2 * 3 || args.length === 3 * 3) {
```

Comment:
With the new API, I'm thinking we'll need to drop `quadraticVertex()` to prevent confusion. If that's the case, maybe it'd be cleaner to stop supporting the old usage of `bezierVertex()` as well? Or is this just temporary while we're doing the refactor?

Reply:
If we keep this around, we can also keep around `quadraticVertex` similarly as a supported but deprecated way of calling this function, as it doesn't prevent the new API from working. This might get moved to the 1.x compatibility addon for a better spot for it too.

Reply:
Okay, whatever makes the most sense! I just posted a few thoughts about this on the 1.x compatibility add-on issue, and I added the p5.js 2.0 label to that issue so that it's easier to find (I guess that label is still appropriate since the purpose of the add-on is to ease the transition to 2.0).